next-steps/index.html

```diff
@@ -28,13 +28,13 @@ <h2>Next Steps</h2>
         <ul class="instructions">
           <li>Go to <a href="https://coverage.nodejs.org/">Node.js Nightly Code Coverage</a>. Find a piece of code that is not covered by tests. After reading the <a href="https://github.com/nodejs/node/blob/master/doc/guides/writing-tests.md">guide to writing tests in the Node.js project</a>, add a new test file or update an existing test file to cover that piece of code.</li>
 
-          <li>Search for <a href="https://github.com/nodejs/node/search?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22&type=Issues">open issues labeled "help wanted"</a> in the issue tracker.</li>
+          <li>Search for <a href="https://github.com/nodejs/node/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22">open issues labeled "help wanted"</a> in the issue tracker.</li>
 
           <li>Search for <a href="https://github.com/nodejs/node/issues?q=is%3Aopen+is%3Aissue+label%3Amentor-available">open issues labeled "mentor-available"</a> in the issue tracker.</li>
 
-          <li>Search for <a href="https://github.com/nodejs/node/search?o=asc&amp;q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22&amp;s=created&amp;state=open&amp;type=Issues&amp;utf8=%E2%9C%93">open issues that are more than a week old and labeled "good first issue"</a> in the issue tracker.</li>
+          <li>Search for <a href="https://github.com/nodejs/node/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22">open issues that are more than a week old and labeled "good first issue"</a> in the issue tracker.</li>
```

Comment:
Probably want to use `+sort%3Acreated-asc` on this link so that the sorting is preserved (oldest issues first, which is important for this one but not the others).

I can add that while landing. Just making a note.